internal/channel/channel.go

```diff
@@ -13,38 +19,88 @@ type Channel struct {
 	Type   string `db:"type"`
 	Config string `db:"config" json:"-"` // excluded from JSON config dump as this may contain sensitive information
 
-	plugin Plugin
+	Logger *logging.Logger
```

Comment:
I don't think there's a reason to export the `Logger` attribute.

Reply:
It is initialised in `config/channel.go`. Must therefore be exported to have access to this property.